networkit/cpp/auxiliary/PQVector.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * PQVector.h
+ *
+ *  Created on: 21.09.2018
+ *      Author: Eugenio Angriman
+ */
+
+#ifndef PQVECTOR_H_
+#define PQVECTOR_H_
+
+#include <algorithm>
+#include <vector>
+
+#include "../auxiliary/Log.h"
+
+namespace Aux {
+/*
+ * A simple prio queue for Kadabra that allows inspecting the ranking.
+ */
+class PQVector {
```

Comment:
This class should have some explanation about why it is useful. Something like:

> This PQ stores at most k elements; if more than k elements are inserted, elements with smallest keys are dropped. The PQ is implemented on top of a vector, thus the insert operation takes O(k) time.

I am not sure if we should call this class really a "priority queue" as it has no `extractMax()`. Maybe `SortedList` would be more appropriate?